src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -45,7 +45,8 @@ ClusterResourceScheduler::ClusterResourceScheduler(
     std::function<bool(scheduling::NodeID)> is_node_available_fn,
     std::function<int64_t(void)> get_used_object_store_memory,
     std::function<bool(void)> get_pull_manager_at_capacity,
-    const absl::flat_hash_map<std::string, std::string> &local_node_labels)
+    const absl::flat_hash_map<std::string, std::string>
+        &local_node_labels)  // pass this to local resource manager
```

Comment:
nit: Seems irrelevant change, just auto-formatter artifact; you may want to revert this change from this PR.